### PR TITLE
Pop window feature

### PR DESCRIPTION
**Pull request by MustafaSamiShirzad**
- [ ] When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- [ ] When the user clicks (or taps) the close (X) button, the popup disappears.
- [ ] You need to use a JavaScript array to store all of the information for all projects.
- [ ] For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- [ ] You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
- [ ] After that, you could implement the popup window.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -607,4 +691,77 @@ footer .section1 .span {
...
+  .popupwindowElement .live_project_btn {
+    width: 70%;
+    height: 50%;
+    padding: 15px 5px;
+  }
+
+  .popupwindowElement .Live_project button img {
+    width: 20px;
+    height: 20px;
+  }
```
Comment (Abenezer-Tilahun) [inline style.css]: - [ ] As stared in the project requirement Kindly make sure **that the popup is responsive, it fits mobile  sizes properly.** As shown below on screenshot the pop up window is not responsive on the mobile screen size 
  
project pop up

- The Figma Pop up window for the mobile screen size look likes this :- 

Pop up project 1

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @MustafaSamiShirzad , 

Great job for implemented  the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linter errors
✔️ Correct Git Hub-Flow
✔️ Good commit messages
✔️ Professional README

almost

## Required Changes ♻️
_Check the comments below the review._

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (MustafaSamiShirzad) [reply style.css]: Thanks for your remarks, I solved the issue and now website looks even more beautifull.

**Review by Cabraham1 — APPROVED** [review]
Hi @MustafaSamiShirzad  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 Impressive UI design ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**